packages/nameguard-sdk/src/index.ts

```diff
@@ -100,7 +103,8 @@ export enum Rating {
 export type SecurePrimaryNameStatus =
   | "normalized" /** The ENS primary name was found and it is normalized. */
   | "no_primary_name" /** The ENS primary name was not found. */
-  | "unnormalized" /** The ENS primary name was found, but it is not normalized. */;
+  | "unnormalized" /** The ENS primary name was found, but it is not normalized. */
+  | "uninspected" /** A name was exceptionally long and was not inspected for performance reasons */;
```

Comment:
@notrab Hey, we've added a new potential `SecurePrimaryNameStatus` here. Can you please investigate and confirm if we need to update anything in `nameguard-react`, `examples.nameguard.io` or `nameguard.io`?